app/build.gradle

```diff
@@ -34,4 +34,5 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    implementation 'com.android.support:recyclerview-v7:28.0.0'
```

Comment:
super annoying... but you want to use this one: implementation 'androidx.recyclerview:recyclerview:1.1.0'

They are technically the same, but the classes have a different namespace/package. I can explain this more later.

Reply:
ok gotcha, just pulled it from the android site. will update.